wire-compiler/src/main/java/com/squareup/wire/model/ProtoTypeName.java

```diff
@@ -45,14 +45,24 @@
   public static final ProtoTypeName UINT32 = new ProtoTypeName("uint32");
   public static final ProtoTypeName UINT64 = new ProtoTypeName("uint64");
 
+  public static final ProtoTypeName FILE_OPTIONS = get("google.protobuf", "FileOptions");
```

Comment:
Yes.

I started by trying to tease apart Java types like `com.squareup.cash.Payment` from Proto types like `squareup.cash.Payment`, and wanted to have a name to identify proto things vs. Java things.

As it turns out, that isn't really important yet, and `Wire` is proto.